solutions/usaco-966.mdx

```diff
@@ -47,4 +47,38 @@ ans += first8[N % 8]
 print(ans)
 ```
 </PySection>
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
+	int n;
+	cin >> n;
+        
+	/*
+	 * Lists the first 8. 
+	 * 14 is in the zeroth position since if were trying to find the 
+	 * eighth number, 8 % 8 = 0.
+	 */
+	vector<int> v{14, 1, 2, 4, 7, 8, 11, 13};
+	
+	int ans = 0;
+	// Finds the group
+	ans = (n - 1) / 8;
+	
+	// Distance away n is from the first 8
```

Comment:
```suggestion
      /*
       * Get the distance away N from the first 8  by calculating (N/8)*15  , since we 
       * have to add for every group of 15.
       */

```